app/next-client-app/app/scanreports/columns.tsx

```diff
@@ -121,44 +124,52 @@ export const columns: ColumnDef<ScanReportList>[] = [
     enableHiding: false,
     cell: ({ row }) => {
       const { id, hidden, dataset } = row.original;
+      const [isOpen, setOpen] = useState<boolean>(false);
 
       return (
-        <DropdownMenu>
-          <DropdownMenuTrigger asChild>
-            <Button variant="ghost" size="sm">
-              <DotsHorizontalIcon className="size-4" aria-hidden="true" />
-            </Button>
-          </DropdownMenuTrigger>
-          <DropdownMenuContent>
-            <DropdownMenuItem
-              onClick={() =>
-                HandleArchive({
-                  id: id,
-                  hidden: hidden,
-                  ObjName: dataset,
-                  type: "scanreports",
-                })
-              }
-            >
-              {hidden ? "Unarchive" : "Archive"}
-              {hidden ? (
-                <EyeOpenIcon className="ml-auto" />
-              ) : (
-                <EyeNoneIcon className="ml-auto" />
-              )}
-            </DropdownMenuItem>
-            <DropdownMenuSeparator />
-            <Link
-              href={`/scanreports/${id}/details/`}
-              style={{ textDecoration: "none", color: "black" }}
-              prefetch={false}
-            >
-              <DropdownMenuItem>
-                Details <Pencil2Icon className="ml-auto" />
+        <>
+          <DropdownMenu>
+            <DropdownMenuTrigger asChild>
+              <Button variant="ghost" size="sm">
+                <DotsHorizontalIcon className="size-4" aria-hidden="true" />
+              </Button>
+            </DropdownMenuTrigger>
+            <DropdownMenuContent>
+              <DropdownMenuItem
+                onClick={() =>
+                  HandleArchive({
+                    id: id,
+                    hidden: hidden,
+                    ObjName: dataset,
+                    type: "scanreports",
+                  })
+                }
+              >
+                {hidden ? "Unarchive" : "Archive"}
+                {hidden ? (
+                  <EyeOpenIcon className="ml-auto" />
+                ) : (
+                  <EyeNoneIcon className="ml-auto" />
+                )}
               </DropdownMenuItem>
-            </Link>
-          </DropdownMenuContent>
-        </DropdownMenu>
+              <DropdownMenuSeparator />
+              <Link
+                href={`/scanreports/${id}/details/`}
+                style={{ textDecoration: "none", color: "black" }}
+                prefetch={false}
+              >
+                <DropdownMenuItem>
+                  Details <Pencil2Icon className="ml-auto" />
+                </DropdownMenuItem>
+              </Link>
+              <DropdownMenuSeparator />
```

Comment:
I don't think we need any of the menu separators here, these are all related enough actions. Could we reorder them though? 
So: Details, Archive, Delete.

(and also change the order on the Dataset table so they are the same, thanks!)